WORKSPACE

```diff
@@ -27,10 +27,10 @@ sass_repositories()
 
 # Add TypeScript rules
 http_archive(
-  name = "build_bazel_rules_typescript",
-  url = "https://github.com/bazelbuild/rules_typescript/archive/0.11.1.zip",
-  strip_prefix = "rules_typescript-0.11.1",
-  sha256 = "7406bea7954e1c906f075115dfa176551a881119f6820b126ea1eacb09f34a1a",
+    name = "build_bazel_rules_typescript",
```

Comment:
remove extra indenting from parameters.

The rest of the file only uses 2 spaces

Reply:
Done.